api/config/config_prod.json

```diff
@@ -33,7 +33,7 @@
   },
   "admin": {
     "adminIdVerification": "workbench-id-verifications@researchallofus.org",
-    "loginUrl": "http:\/\/all-of-us-rw-prod.appspot.com/login"
+    "loginUrl": "http:\/\/workbench.researchallofus.org/login"
```

Comment:
https? the http link doesn't work for me.

Reply:
Nice catch - seems like all of them should be using https (except local). Made this change across the board.